.github/workflows/main.yml

```diff
@@ -65,4 +65,5 @@ jobs:
       
       - name: Execute benchmarking
         run: |
+          pip install scanorama
```

Comment:
this is because we run on a different runner with a different user in a different environment